docs/content/commands/rclone.md

```diff
@@ -123,7 +123,7 @@ rclone [flags]
       --box-token-url string                                Token server url
       --box-upload-cutoff SizeSuffix                        Cutoff for switching to multipart upload (>= 50 MiB) (default 50Mi)
       --buffer-size SizeSuffix                              In memory buffer size when reading files for each --transfer (default 16Mi)
-      --bwlimit BwTimetable                                 Bandwidth limit in KiB/s, or use suffix B|K|M|G|T|P or a full timetable
+      --bwlimit BwTimetable                                 Bandwidth limit in KiB/s, or use suffix B|K|M|G|T|P or a full timetable. Entries can be separated by spaces or semicolons.
```

Comment:
Can you drop this change - `docs/content/commands/rclone.md` is autogenerated (it says at the top) and I don't think we need the command line help getting any longer.

Reply:
Oops, that's my mistake. fixed now.